tensorflow/workspace.bzl

```diff
@@ -60,6 +60,11 @@ def check_bazel_version_at_least(minimum_bazel_version):
     fail("\nCurrent Bazel version is {}, expected at least {}\n".format(
         native.bazel_version, minimum_bazel_version))
 
+# Sanitize a dependency so that it works correctly from code that includes
+# TensorFlow as a submodule.
+def clean_dep(dep):
```

Comment:
Let's either modify the whole file, or split this out of this PR as a separate PR.
Just want to make sure the file is consistent.

Reply:
Sure. I will update all occurrences, if that is acceptable =)

Reply:
Done